cloudbuild.yaml

```diff
@@ -7,5 +7,6 @@ steps:
       - PULL_BASE_REF=${_PULL_BASE_REF}
       - REGISTRY_NAME=gcr.io/${_STAGING_PROJECT}
       - HOME=/root
+    timeout: 1200s
```

Comment:
the overall timeout is already 1200s (see first line of this file) so based on doc this shouldn't be necessary: https://cloud.google.com/cloud-build/docs/build-config